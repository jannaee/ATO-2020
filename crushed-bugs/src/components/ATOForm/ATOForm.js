import { 
    Form, 
    TextArea, 
    TextInput, 
    Select, 
    Button } from 'carbon-components-react';
import React from 'react';

const ATOForm = () => {
    return (
        <Form className="someForm">
            <div style={{ marginBottom: '2rem' }}>
                <TextInput
                    id="test2"
                    invalidText="Invalid error message."
                    labelText="First input field"
                    placeholder="Placeholder text"
                />
            </div>
            <div style={{ marginBottom: '2rem' }}>
                <TextArea
                    cols={50}
                    // helperText="Optional helper text here; if message is more than one line text should wrap (~100 character count maximum)"
                    id="test5"
                    invalidText="Invalid error message."
                    labelText="Second input field"
                    placeholder="Placeholder text"
                    rows={4}
                />
            </div>
            <div style={{ marginBottom: '2rem' }}>
                <Select
                    defaultValue="placeholder-item"
                    id="select-1"
                    invalidText="This is an invalid error message."
                    labelText="Dropdown"
                >
                </Select>
            </div>
            <Button
                kind="primary"
                tabIndex={0}
                type="submit"
            > Submit
            </Button>
        </Form>

    )
}


export default ATOForm;