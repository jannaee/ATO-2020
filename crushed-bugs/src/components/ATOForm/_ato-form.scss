.title {
    margin-bottom: 3rem;
    color: black;
    font-weight: bold;
}