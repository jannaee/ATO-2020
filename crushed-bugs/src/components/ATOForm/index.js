import ATOForm from './ATOForm';
export default ATOForm;