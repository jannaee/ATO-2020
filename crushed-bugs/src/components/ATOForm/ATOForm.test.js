import React from 'react';
import { shallow } from 'enzyme';
import ATOForm from './ATOForm';

const title = 'Test Title';
let wrapper = shallow(<ATOForm>{title}</ATOForm>);

describe('ATOForm title', () => {
    it('should render the ATOForm Component correctly', () => {   
        expect(wrapper).toMatchSnapshot();
    });
});