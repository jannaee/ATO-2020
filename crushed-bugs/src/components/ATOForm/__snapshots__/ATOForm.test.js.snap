// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ATOForm /> rendering should render 1 form   1`] = `""`;
